action.yml

```diff
@@ -2,52 +2,51 @@ name: Update Deployment Board
 description: Adds deployment Issues to an Automated Deployment Project Board for visual tracking of what has been deployed to each environment.
 
 inputs:
-  github-token:
-    description: 'A token with permissions to create and update issues'
+  workflow-actor:
+    description: The GitHub user who triggered the workflow
     required: true
-  github-login:
-    description: 'The login associated with the github-token.  Defaults to github-actions'
+  token:
+    description: A token with permissions to create and update issues
     required: true
   environment:
-    description: 'The environment the branch, tag or SHA was deployed to'
+    description: The environment the branch, tag or SHA was deployed to, i.e. [DEV|QA|STAGE|DEMO|UAT|PROD]
     required: true
-  board-number:
-    description: 'The number of the project board that will be updated.  Found by using the number in the board url.'
+  project-slug:
+    description: The repo the deployment will be created in, i.e. [org|owner]/[repo name]
     required: true
   ref:
-    description: 'The branch, tag or SHA that was deployed'
+    description: The branch, tag or SHA that was deployed
     required: true
```

Comment:
`ref` seems so nondescript. I'd like to be able to say `release` or `version`, but I know not everything is going to deploy from an actual GitHub "release object". Some things are going to deploy from a tag without a release, and some things may even deploy from a branch without a tag.

Could we change this input to `release_version`, `version_ref_name` or `release_ref`?